File: 08_siguientePrimo/C/sigPrimo.c
#include<stdio.h>
int j;
int primo(int k)
{
    int i;
    for(i=2;i<k;i++)
    {
        if(k%i==0)
            return 0;
    }
    return 1;
}
int main()
{
    printf("Ingresa un numero: ");
    scanf("%d",&j);
    j++;
    while(primo(j)==0)
        j++;
    printf("El siguiente primo es: %d\n",j);
    return 0;
}

File: 08_siguientePrimo/Python/sigPrimo.py
def primo(k):
    for i in range(2,k):
        if int(k%i)==0:
            return False
    return True
n=int(input("Ingresa un numero: "))
n=n+1
while not primo(n):
    n=n+1
print("El siguiente primo es: %d"%(n))